src/main/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/BankHolidaysClient.java

```diff
@@ -1,19 +1,29 @@
 package uk.gov.hmcts.reform.sendletter.services.date.holidays;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 import org.springframework.web.client.RestTemplate;
 import uk.gov.hmcts.reform.sendletter.services.date.holidays.response.Holidays;
 
+@Component
 public class BankHolidaysClient {
 
+    private static final String DEFAULT_URL = "https://www.gov.uk/bank-holidays/england-and-wales.json";
```

Comment:
I'd still recommend taking it from configuration, as that's where one would search for it.

Reply:
taking a cue from gov notify client which has url hardcoded

Reply:
We include our dependencies' URLs in the config by convention. This one makes an exception now.